rs/nns/test_utils/src/governance.rs

```diff
@@ -356,55 +291,6 @@ pub fn bump_gzip_timestamp(wasm: &Wasm) -> Wasm {
     Wasm::from_bytes(new_wasm)
 }
 
-/// Submits a proposal to upgrade the root canister.
-pub async fn upgrade_root_canister_by_proposal(
```

Comment:
This is no longer needed since `InstallCode` works for both root and non-root canisters.